yarn.lock

```diff
@@ -8600,6 +8600,11 @@ yargs@^15.3.1:
     y18n "^4.0.0"
     yargs-parser "^18.1.1"
 
+yarn@^1.22.4:
```

Comment:
Wonder where this is coming from?

Reply:
Me too, just ran `yarn` in the top level.

Reply:
Looks like it's coming from adding `yarn` to `package.json`.